auth-web/src/util/constants.ts

```diff
@@ -147,7 +147,7 @@ export enum InvoiceStatus {
 
 export enum AffiliationTypes {
     NAME_REQUEST = 'Name Request',
-    AMALGAMATION = 'Amalgamation Application',
+    AMALGAMATION_APPLICATION = 'Amalgamation Application',
     INCORPORATION_APPLICATION = 'Incorporation Application',
     CORPORATION = 'Corporation',
     REGISTRATION = 'Registration'
```

Comment:
Ideally, you'd import the shared FilingNames enum, and then lines 150, 151 and 153 would use it, similar to this:
https://github.com/bcgov/namerequest/blob/43a847d902256016a6d40e15e9066b6ea38f2d9a/src/enums/entity-types.ts#L6

I know this will result in a mix of references and strings, but I favour having things declared in a single place.